R/extract_ngram.R

```diff
@@ -9,9 +9,37 @@
 #'
 #' @examples
 #' tweets <- c(
-#'    "Make America Great Again DonaldTrump"
+#'    "Make", "America", "Great", "Again", "DonaldTrump"
 #' )
 #' extract_ngram(tweets, n=3)
 extract_ngram <- function(tweets, n = 3) {
+  if (length(tweets) < n) {
+    stop("length of ngrams should be less than length of tweets")
```

Comment:
Shouldn't say "less than # of words in vector of tweets"?

Reply:
Thanks. Updated the error message.